modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -309,6 +334,39 @@ private JSONObject _getClipboardJSONObject(
 			});
 	}
 
+	private boolean _isSharingEnabled(
+		long classNameId, long classPK, ContentDashboardItem<?> contentDashboardItem,
+		ThemeDisplay themeDisplay ) {
+
+		SharingConfiguration sharingConfiguration =
```

Comment:
You should read the company segmentation too

Reply:
Done! thank you!